azure-spring-boot/src/main/resources/microsoftGraphServiceEndpoints.properties

```diff
@@ -0,0 +1,9 @@
+azure.service.endpoints.cn-microsoftgraph.aadSigninUri=https://login.partner.microsoftonline.cn/
+azure.service.endpoints.cn-microsoftgraph.aadGraphApiUri=https://microsoftgraph.chinacloudapi.cn/
+azure.service.endpoints.cn-microsoftgraph.aadKeyDiscoveryUri=https://login.partner.microsoftonline.cn/common/discovery/keys
+azure.service.endpoints.cn-microsoftgraph.aadMembershipRestUri=https://microsoftgraph.chinacloudapi.cn/v1.0/me/memberOf
+azure.service.endpoints.global-microsoftgraph.aadSigninUri=https://login.microsoftonline.com/
+azure.service.endpoints.global-microsoftgraph.aadGraphApiUri=https://graph.microsoft.com/
+azure.service.endpoints.global-microsoftgraph.aadKeyDiscoveryUri=https://login.microsoftonline.com/common/discovery/keys/
+azure.service.endpoints.global-microsoftgraph.aadMembershipRestUri=https://graph.microsoft.com/v1.0/me/memberOf
+azure.service.endpoints.global-microsoftgraph.aadMicrosoftGraphApiBool=true
```

Comment:
just put these endpoints in the file serviceEndpoints.properties. There is no need to separate them into two files.